src/services/LoggingService.js

```diff
@@ -3,36 +3,67 @@
  *
  * Requires the NewRelic Browser JavaScript snippet.
  */
+
+// NewRelic will not log an error if it is too long.
+const MAX_ERROR_LENGTH = 4000;
```

Comment:
@abutterworth @georgebabey: FYI: I left this in because I think it is better to have some of the message than none of the message.  However, I did learn that the event is logged either way (with the newer newrelic snippet and possibly with the older one), but the default view is filtered on message and the message itself is not sent so the error is filtered from that view.  If you switch to `errorClass`, then you might be able to see the additional errors.